src/test/scala/cromwell/engine/db/slick/SlickDataAccessSpec.scala

```diff
@@ -36,22 +38,34 @@ class SlickDataAccessSpec extends FlatSpec with Matchers with ScalaFutures {
 
   object UnknownBackend extends Backend {
     type BackendCall = LocalBackendCall
-    override def adjustInputPaths(call: Call, inputs: CallInputs) = Map.empty
-    override def adjustOutputPaths(call: Call, outputs: CallOutputs): CallOutputs = outputs
-    override def stdoutStderr(workflowId: WorkflowId, workflowName: String, callName: String): StdoutStderr = ???
 
-    override def initializeForWorkflow(workflow: WorkflowDescriptor) = Map.empty
+    override def adjustInputPaths(call: Call, inputs: CallInputs) =
+      throw new NotImplementedError
```

Comment:
I find `???` to be easier for me to notice that something's not implemented just because it's more visually jarring